Preparing training and test sets...
Creating samples using as look_back = 15
Traceback (most recent call last):
  File "/Applications/PyCharm.app/Contents/helpers/pydev/pydev_run_in_console.py", line 52, in run_file
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Applications/PyCharm.app/Contents/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/michele/Fraud_Detection_LSTM_Thesis/LSTM_classifier.py", line 127, in <module>
    x, y, transactions = create_dataset(bonifici, 15)
  File "/Users/michele/Fraud_Detection_LSTM_Thesis/LSTM_classifier.py", line 74, in create_dataset
    t.extend(t)
  File "/Users/michele/PycharmProjects/varie/venv/lib/python3.6/site-packages/pandas/core/generic.py", line 5179, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'extend'
PyDev console: starting.

Python 3.6.3 (v3.6.3:2c5fed86e0, Oct  3 2017, 00:32:08) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
